### DISCUSSION TITLE
Wich to use? WindowsAppSDK or Project Reunion

### DISCUSSION THREAD

**Discussion by ombolics** [microsoft/WindowsAppSDK#2175] [Q&A] [answered]
Hi,
I can't figure out, wich nuget package should I install. 
![image](https://user-images.githubusercontent.com/56260703/155718816-95bf3e34-30d0-466f-b3b2-58ac32e33638.png)
![image](https://user-images.githubusercontent.com/56260703/155719048-745819ad-96b3-46fe-8b34-8f33f81ec8e1.png)

The latest published version of Project Reunion is newer than the WinApp SDK's latest published version. What is the difference between these two?
Thank you for any help!

**Comment by hez2010** [comment, marked as answer]
Use WindowsAppSDK 1.0.
The 0.8.6 ProjectReuion package is just a serving upgrade for existing old version users.
